napari/layers/image/image.py

```diff
@@ -322,16 +370,74 @@ def _set_view_slice(self):
         else:
             order = self.dims.displayed_order
 
-        image = np.asarray(self.data[self.dims.indices]).transpose(order)
+        if self.pyramid:
+            # If 3d redering just show lowest level of pyramid
+            if self.dims.ndisplay == 3:
+                self.data_level = len(self._data_pyramid) - 1
+
+            # Slice currently viewed level
+            level = self.data_level
+            indices = np.array(self.dims.indices)
+            downsampled = indices[nd] / self.level_downsamples[level, nd]
```

Comment:
Maybe call this `downsampled_indices`, `downsampled_idxs`, or `curr_level_indices`, or similar? Downsampled is the name we gave to downsampled images elsewhere, so this whole section was super-confusing to me for a bit.

Reply:
yeah, agreed - changed to `downsampled_indices`